radixdlt-core/radixdlt/src/main/java/com/radixdlt/consensus/LedgerHeader.java

```diff
@@ -196,7 +198,7 @@ public JSONObject asJSONObject(Addressing addressing) {
 
 
 	public static LedgerHeader mocked() {
-		return new LedgerHeader(0, View.genesis(), new AccumulatorState(0,  HashUtils.zero256()), 0, null);
+		return new LedgerHeader(0, View.genesis(), new AccumulatorState(0, HashUtils.zero256()), 0, null);
```

Comment:
Is it used fot tests? If so, don't believe it should be here. Maybe a TODO to remove that?

Reply:
added TODO

Reply:
added TODO and created ticket